qiskit_ibm_runtime/options/resilience_options.py

```diff
@@ -53,7 +49,7 @@ class ResilienceOptions:
             Default: ``None``, and ``LinearExtrapolator`` if resilience level is 2.
     """
 
-    noise_amplifier: NoiseAmplifierType = None
+    noise_amplifier: NoiseAmplifierType = "LocalFoldingAmplifier"
```

Comment:
We can't merge this until there is an update on the server side right? Because this would add `noise_amplifier` to every job, including those without `resilience_level` 2

Reply:
As far as I understand, this will have no effect on other resilience levels, because ZNE won't be run. @jyu00 - any comment on this?